src/Microsoft.Data.Analysis/StringDataFrameColumn.BinaryOperations.cs

```diff
@@ -128,6 +131,26 @@ internal static PrimitiveDataFrameColumn<bool> ElementwiseNotEqualsImplementatio
             return ret;
         }
 
+        public override PrimitiveDataFrameColumn<bool> ElementwiseIsNotNull()
+        {
+            PrimitiveDataFrameColumn<bool> ret = new PrimitiveDataFrameColumn<bool>(Name, Length);
+            for (long i = 0; i < Length; i++)
+            {
+                ret[i] = this[i] != null;
```

Comment:
please use IsValid

Reply:
We are not able to use IsValid here, as StringDataFrameColumn doesn't inherit from PrimitiveDataFrameColumn class where IsValid method is defiined. It doesn't have ValidyBuffer and stores list of objects instead of an instance of PrimitiveColumnContainer, so check for null seems to be safe here